MANIFEST.in

```diff
@@ -1,5 +1,10 @@
 include AUTHORS
 include ChangeLog
+
+recursive-include horizon_bsn/bsndashboard/templates *
+recursive-include horizon_bsn/bsndashboard/static *
+recursive-include horizon_bsn/bsndashboard/reachabilitytests/templates *
```

Comment:
i think i added these previously when things didn't work. can you retry by removing these and see if it still works?
we may not need to recursively include this.